build/travis/job2_AppImage/build.sh

```diff
@@ -27,17 +29,31 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build MuseScore AppImage inside Docker image
-docker run -i -v "${PWD}:/MuseScore" library/centos:6 \
-   /bin/bash -c "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+# Build AppImage. Are we cross-compiling?
+case "$1" in
+  --32bit )
+    shift
+    # Build MuseScore AppImage inside 32-bit Docker image
+    docker run -i -v "${PWD}:/MuseScore" toopher/centos-i386:centos6 /bin/bash -c \
+      "linux32 --32bit i386 /MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+  * )
+    [ "$1" == "--64bit" ] && shift || true
+    # Build MuseScore AppImage inside native (64-bit) Docker image
+    docker run -i -v "${PWD}:/MuseScore" library/centos:6 /bin/bash -c \
+      "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+esac
```

Comment:
@ericfont If you want to cross-compile for ARM on Travis this is the place to do it. Add a new case `--arm` for the ARM-specific build command. You can then call it from .travis.yml like this: `build.sh --arm` ([see how the 32 bit works](https://github.com/musescore/MuseScore/pull/2398/files#diff-354f30a63fb0907d4ad57269548329e3R83)). The best way to cross-compile is to enter a chroot so that you can avoid conflicts with the native system's libraries when you run copy-libs, which could otherwise end up fetching x86_64 libraries instead of ARM ones.

I haven't fully looked into this myself, but it looks like you need to install a [static QEMU](https://wiki.ubuntu.com/ARM/BuildEABIChroot) binary, and then enter the chroot somehow, possibly using AppImageKit or Docker. There are a few ARM-based Docker images:
Search for "rpi" or "arm" here: https://hub.docker.com
Or here's an Arch linux image: https://hub.docker.com/r/cellofellow/rpi-arch/

I'm not sure if it will work with Docker, but [here's how another guy did something similar](https://www.tomaz.me/2013/12/02/running-travis-ci-tests-on-arm.html).

Or I may have a look into this myself at some point if you don't fancy it.